hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/security/symmetric/TestLocalKeyStore.java

```diff
@@ -70,13 +71,13 @@ void setup() throws IOException {
   static Stream<Arguments> saveAndLoadTestCases() throws Exception {
     return Stream.of(
         // empty
-        Arguments.of(ImmutableList.of()),
+        of(ImmutableList.of()),
         // single secret keys.
-        Arguments.of(newArrayList(
+        of(newArrayList(
             generateKey("HmacSHA256")
         )),
         // multiple secret keys.
-        Arguments.of(newArrayList(
+        of(newArrayList(
```

Comment:
Let's keep `Arguments.of`, since `of` is very generic.  For example we also have `Stream.of` and `ImmutableList.of` in this short method.

Reply:
Thanks for review! fix comments.